backend/src/nodes/impl/pytorch/torch_types.py

```diff
@@ -6,9 +6,9 @@
 from .architecture.SwinIR import SwinIR
 from .architecture.Swin2SR import Swin2SR
 from .architecture.HAT import HAT
-from .architecture.face_archs.gfpganv1_clean_arch import GFPGANv1Clean
-from .architecture.face_archs.restoreformer_arch import RestoreFormer
-from .architecture.face_archs.codeformer import CodeFormer
+from .architecture.face_arches.gfpganv1_clean_arch import GFPGANv1Clean
```

Comment:
Maybe we could just call it `face`? We already know that it's about architectures.

```suggestion
from .architecture.face.gfpganv1_clean_arch import GFPGANv1Clean
```